crdt_commands_test.go

```diff
@@ -1245,3 +1245,287 @@ func TestValidationOfFetchMapViaBuilder(t *testing.T) {
 		t.Errorf("expected %v, actual %v", expected, actual)
 	}
 }
+
+// UpdateHll
+// DtUpdateReq
+// DtUpdateResp
+
+func TestBuildDtUpdateReqCorrectlyViaUpdateHllCommandBuilder(t *testing.T) {
+	builder := NewUpdateHllCommandBuilder().
+		WithBucketType("hlls").
+		WithBucket("bucket").
+		WithKey("key").
+		WithAdditions([]byte("a1"), []byte("a2")).
+		WithAdditions([]byte("a3"), []byte("a4")).
+		WithW(1).
+		WithDw(2).
+		WithPw(3).
+		WithReturnBody(true).
+		WithTimeout(time.Second * 20)
+	cmd, err := builder.Build()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+
+	if _, ok := cmd.(retryableCommand); !ok {
+		t.Errorf("got %v, want cmd %s to implement retryableCommand", ok, reflect.TypeOf(cmd))
+	}
+
+	protobuf, err := cmd.constructPbRequest()
+	if err != nil {
+		t.Fatal(err.Error())
+	}
+	if protobuf == nil {
+		t.Fatal("protobuf is nil")
+	}
+	if req, ok := protobuf.(*rpbRiakDT.DtUpdateReq); ok {
+		if expected, actual := "hlls", string(req.GetType()); expected != actual {
```

Comment:
So this is super pedantic... I've been slowly converting the tests to the golang "standard" of `got, want`:

https://github.com/basho/riak-go-client/blob/master/command_rpb_test.go#L19-L25

New tests should follow this I suppose.